scene/gui/gradient_edit.h

```diff
@@ -45,6 +45,10 @@ class GradientEdit : public Control {
 	bool grabbing = false;
 	int grabbed = -1;
 	Vector<Gradient::Point> points;
+	Gradient::InterpolationMode interpolation_mode = Gradient::GRADIENT_INTERPOLATE_LINEAR;
+
+	Ref<Gradient> gradient_cache;
+	Ref<GradientTexture> preview_texture;
```

Comment:
Since https://github.com/godotengine/godot/pull/54715 was merged I think it should be renamed to `GradientTexture1D`

Reply:
Rebased.